src/chrome/content/rules/Socialblade.com-falsemixed.xml

```diff
@@ -1,13 +0,0 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://forums.socialblade.com/ => https://forums.socialblade.com/: (6, 'Could not resolve host: forums.socialblade.com')
-
-	See also Socialblade.com.xml
--->
-<ruleset name="Socialblade.com (mixed content)" platform="mixedcontent" default_off='failed ruleset test'>
```

Comment:
`forums` is down, but `socialblade.com` itself is still up (see `Socialblade.com.xml`).